ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-aro-4.12.yaml

```diff
@@ -30,6 +30,12 @@ releases:
       product: ocp
       stream: nightly
       version: "4.12"
+  source:
```

Comment:
`openshift-extended-upgrade-pre` is not referenced in `openshift-upgrade-qe-test-e2e`, so we do not need the setting here.